src/components/Nav/Nav.js

```diff
@@ -1,10 +1,113 @@
 import React from 'react';
+import { withRouter } from 'react-router-dom';
+import { Link } from 'react-router-dom';
```

Comment:
```suggestion
import { withRouter, Link } from 'react-router-dom';
```
이런식으로 축약해서 사용할 수 있습니다!! 결국 객체니까요!

Reply:
오 도현님 방법을 제시해주셔서 너무 감사드립니다! 바로 수정 해 보겠습니다!🤭